components/builder-db/src/models/jobs.rs

```diff
@@ -1,1120 +0,0 @@
-use super::db_id_format;
```

Comment:
Similar to above, drop these db items now or wait for a later time and a different PR.

Reply:
DB cleanup in terms of dropping unused items needs to happen in some future PR as this one is huge.